website/src/pages/dictionary/index.js

```diff
@@ -117,6 +136,46 @@ function DataDictionary() {
     );
   };
 
+  // TODO: Memo
+  const searchSchemas = () =>
```

Comment:
this is only used once, we can throw this inline in `filteredSchemas` definition directly to avoids having to jump around to read.